HPXMLtoOpenStudio/tests/test_miscloads.rb

```diff
@@ -36,6 +47,50 @@ def test_misc_loads
     # Check television
     tv_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscTelevision)
     assert_in_delta(619, tv_kwh_yr, 1.0)
+
+    # Check vehicle
+    vehicle_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscVehicle)
+    assert_in_delta(100, vehicle_kwh_yr, 1.0)
+
+    # Check well pump
+    wp_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscWellPump)
+    assert_in_delta(100, wp_kwh_yr, 1.0)
+
+    # Check pool pump
+    pp_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscPoolPump)
+    assert_in_delta(50, pp_kwh_yr, 1.0)
+
+    # Check pool electric heater
+    ph_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscPoolHeater)
+    assert_in_delta(100, ph_kwh_yr, 1.0)
+
+    # Check pool gas heater
+    ph_therm_yr = get_therm_per_year(model, Constants.ObjectNameMiscPoolHeater)
+    assert_in_delta(100, ph_therm_yr, 1.0)
+
+    # Check hot tub pump
+    htp_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscHotTubPump)
+    assert_in_delta(50, htp_kwh_yr, 1.0)
+
+    # Check hot tub electric heater
+    hth_kwh_yr = get_kwh_per_year(model, Constants.ObjectNameMiscHotTubHeater)
+    assert_in_delta(100, hth_kwh_yr, 1.0)
+
+    # Check hot tub gas heater
+    hth_therm_yr = get_therm_per_year(model, Constants.ObjectNameMiscHotTubHeater)
+    assert_in_delta(100, hth_therm_yr, 1.0)
+
+    # Check gas grill
+    gg_therm_yr = get_therm_per_year(model, Constants.ObjectNameMiscGasGrill)
+    assert_in_delta(10, gg_therm_yr, 1.0)
+
+    # Check gas lighting
+    gl_therm_yr = get_therm_per_year(model, Constants.ObjectNameMiscGasLighting)
+    assert_in_delta(10, gl_therm_yr, 1.0)
+
+    # Check gas fireplace
+    gf_therm_yr = get_therm_per_year(model, Constants.ObjectNameMiscGasFireplace)
+    assert_in_delta(10, gf_therm_yr, 1.0)
```

Comment:
Some of these values in the sample file seem unreasonable. Can you look at the default values for each of these misc loads and choose similar (but not identical) values?

Reply:
Also, looks like similar tests are needed in test_appliances.rb for the extra refrigerator and freezer.

Reply:
Good catch.